scripts/release_ppa.sh

```diff
@@ -57,7 +57,7 @@ packagename=solc
 
 static_build_distribution=focal
 
-DISTRIBUTIONS="bionic eoan focal"
+DISTRIBUTIONS="bionic focal"
```

Comment:
Aren't there new ones we should add?

Reply:
The next one is Groovy Gorilla (https://en.wikipedia.org/wiki/Ubuntu_version_history#Table_of_versions). It will be out on  2020-10-22. It's already on Launchpad as the development version (https://launchpad.net/ubuntu/groovy) so I guess it should be possible to add it. But I'm not sure if it's a good idea if we're not testing it in CI.

Reply:
We can test compilation via the dev releases and if it fails for some other reason, we have different problems :)

Reply:
Well, we generally don't test all of them in CI... i.e. there was never a test for ``eoan`` either. But yes, we can add it here and I'll also add it to the static z3 deps build (and run it for groovy as well).
But yeah - the october versions are generally very short-lived, I'm not sure it's worth setting up CI infrastructure for them (and apparently we haven't done it in the past) - but having PPA releases for them sounds good.

Reply:
I just triggered a "groovy" build for libz3-static-dev and launchpad indeed accepted it, so I also added it here.